treeherder/log_parser/utils.py

```diff
@@ -29,14 +29,12 @@ def extract_text_log_artifacts(project, log_url, job_guid):
     return artifact_list
 
 
-def post_log_artifacts(project, job_guid, job_log):
+def post_log_artifacts(job_log):
     """Post a list of artifacts to a job."""
-    log_url = job_log.url
-    log_description = "%s %s (%s)" % (project, job_guid, log_url)
```

Comment:
I don't think that's a big problem (as you say the same applies to NR, which is almost always the first port of call and by the time you are looking at papertrail, it's likely that you'll have a db session open anyway).